src/app/(dashboard)/dashboard/page.tsx

```diff
@@ -0,0 +1,23 @@
+import ChallengesCalendar from '@/components/challenges-calendar';
+import ChallengesCompletion from '@/components/challenges-completion';
+import UserLevel from '@/components/user-level';
+
+const Home = () => {
+  return (
+    <main className='mx-auto mt-[-48px] flex h-full max-w-7xl flex-col justify-center px-6'>
```

Comment:
This negative margin is necessary?

Reply:
fixed at commit 
- [fix: replaced negative margin-top with positive padding-bottom](https://github.com/Super-Patos-Hackaton/frontend/commit/28ff626214a13fde1994482f0ad2002d10c24d41)